src/app/showcase/components/multiselect/multiselectdemo.html

```diff
@@ -27,7 +27,7 @@ <h3>Templating</h3>
     <p>Selected Cars: {{selectedCars2}}</p>
 
     <h3>Virtual Scroll (10000 Items)</h3>
-    <p-multiSelect [options]="items" [(ngModel)]="item" [panelStyle]="{minWidth:'12em'}" [virtualScroll]="true" [itemSize]="34" [filter]="false"></p-multiSelect>
+    <p-multiSelect [options]="items" [(ngModel)]="item" [panelStyle]="{minWidth:'12em'}" [virtualScroll]="true" [filter]="true" [itemSize]="34" [filter]="false"></p-multiSelect>
```

Comment:
```
[filter]="true" [itemSize]="34" [filter]="false"
```
That seems odd.

Reply:
@Mysame  Agree, honest mistake, should we create a new issue for it?

First `filter` input currently overrides the second one.

Reply:
You probably should, as of now it isn't clear and looks wrong (even if it functions correctly)

Reply:
#7339